runtime/moonbeam/src/lib.rs

```diff
@@ -1524,6 +1524,7 @@ pub type SignedExtra = (
 	frame_system::CheckNonce<Runtime>,
 	frame_system::CheckWeight<Runtime>,
 	pallet_transaction_payment::ChargeTransactionPayment<Runtime>,
+	cumulus_primitives_storage_weight_reclaim::StorageWeightReclaim<Runtime>,
```

Comment:
We should not use StorageWeightReclaim as it's not compatible with ethereum transactions, please discard this part

Reply:
Removed it, but maybe there is a way to make frontier compatible with this.